### DISCUSSION TITLE
When using gpg signing, `git commit` is shown fullscreen

### DISCUSSION THREAD

**Discussion by peppy** [jesseduffield/lazygit#3055] [Q&A]
If adding

```
[commit]
	gpgsign = true
```

to one's `.gitconfig`, using the "quick commit" dialog modal will result in a fullscreen view of the `git commit` output  after running, and require an extra enter keypress. Is this a technical limitation in some way?

without:

https://github.com/jesseduffield/lazygit/assets/191335/8a3c558a-d317-4a80-9d86-34174039b84c

with:

https://github.com/jesseduffield/lazygit/assets/191335/5b2d95ab-47c3-48eb-8dfb-b01f05639b05

**Comment by peppy** [comment]
Given that many other operations aren't blocked by this (ie. I can successfully reorder commits), I believe this extra prompt can be avoided. Pretty large usability issue for me at the moment. I may dive into the code myself and see what I can come up with, but interested in any historical context which can be added here by those that know more than me.

I did see that some operations are hard blocked with GPG signing is turned on as well, but given on one's setup it shouldn't block any operation (in the case that you are using SSH signing keys with an `ssh-agent` meaning no passkey entry, for instance).

**Reply by nicolandu** [reply]
The same goes for me with SSH signing enabled.
